test/built/regression/generateRegressionTests.js.md

```diff
@@ -0,0 +1,222 @@
+# Snapshot report for `built/regression/generateRegressionTests.js`
```

Comment:
This works for now, but I think we really need to figure out how to split the results amongst multiple files.

Reply:
I'll dig into Ava's source and see if there's a way to over-ride the test context's attributes and trick it into thinking that the test is located somewhere that it isn't.